server/src/main/java/org/elasticsearch/action/support/IndicesOptions.java

```diff
@@ -324,14 +303,14 @@ public static Builder builder(WildcardOptions wildcardOptions) {
      * - The ignoreThrottled flag, which is a deprecated flag that will filter out frozen indices.
      * @param allowAliasToMultipleIndices, allow aliases to multiple indices, true by default.
      * @param allowClosedIndices, allow closed indices, true by default.
-     * @param allowFailureIndices, allow failure indices in the response, true by default
```

Comment:
I put up https://github.com/elastic/elasticsearch/pull/118614/commits/2fd1abac3cccf36fe50d2c548b7f8ea91f4d3293 as an alternative to doing this in the resolver. Happy to talk about changing it though if we think this restriction should be available to more places in the code.

Reply:
Hm, I see, I was thinking of modelling it like the closed indices but I see the value of not adding another flag.

From a quick thought, the APIs I had in mind are the ones that edit the contents of the failure index, such as update/delete by query.

Another one that I had in mind is the `PUT <target>/_settings`, but I wasn't sure about this.

Reply:
I want to avoid locking down `PUT _settings` too much in case we need to provide work arounds for end user support. I don't think update/delete by query needs to be blocked from happening on failure indices per se, but if we decide on blocking those operations for them then I don't think it'll be too hard to do in a follow up.